services/gateway/sources/mappings/peer.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
 	os: '=,string',
 	version: '=,string',
 	state: '=,number',
+	stateName: '=,string',
```

Comment:
@sameersubudhi @nagdahimanshu? `stateName` or `stateText`? What do you think?

Reply:
I prefer `stateName` for now.
If it weren't for the backward compatibility, I would prefer something like `connectionState` (text) and `connectionStateCode` (number).

Reply:
Right, I'll keep this in mind for the next API design.